scripts/install-forms-flow.sh

```diff
@@ -2,131 +2,150 @@
+        version_ff_ai="latest"
+        version_ff_analytics="latest"
+        version_ff_api="latest"
+        version_ff_bpm="latest"
+        version_ff_data_analysis="latest"
+        version_ff_documents_api="latest"
+        version_ff_forms="latest"
+        version_ff_idm="latest"
+        version_ff_web="latest"
+    else
+        # See https://github.com/AOT-Technologies/forms-flow-ai-charts/releases
+        version_ff_admin="v7.1.1"
+        version_ff_ai="v7.1.1"
+        version_ff_analytics="v7.1.1"
+        version_ff_api="v7.1.1"
+        version_ff_bpm="v7.1.1"
+        version_ff_data_analysis="v7.1.1"
+        version_ff_documents_api="v7.1.1"
+        version_ff_forms="v7.1.1"
+        version_ff_idm="v7.1.1"
+        version_ff_web="v7.1.1"
+    fi
 }
 
 addRegistry() {
-	helm repo remove formsflow
-	helm repo add formsflow https://aot-technologies.github.io/forms-flow-ai-charts
-	echo
-	helm repo update formsflow
+    helm repo remove formsflow
+    helm repo add formsflow https://aot-technologies.github.io/forms-flow-ai-charts
+    echo
+    helm repo update formsflow
 }
 
 updateLocalDependencies(){
-	helm dependency update $directory/forms-flow-ai/
-	helm dependency update $directory/forms-flow-analytics/
-	helm dependency update $directory/forms-flow-forms/
-	helm dependency update $directory/forms-flow-idm/
-	helm dependency update $directory/forms-flow-admin/
-	helm dependency update $directory/forms-flow-api/
-	helm dependency update $directory/forms-flow-bpm/
-	helm dependency update $directory/forms-flow-data-analysis/
-	helm dependency update $directory/forms-flow-web/
-	helm dependency update $directory/forms-flow-documents-api/
+    helm dependency build $directory/forms-flow-ai/
+    helm dependency build $directory/forms-flow-analytics/
+    helm dependency build $directory/forms-flow-forms/
+    helm dependency build $directory/forms-flow-idm/
+    helm dependency build $directory/forms-flow-admin/
+    helm dependency build $directory/forms-flow-api/
+    helm dependency build $directory/forms-flow-bpm/
+    helm dependency build $directory/forms-flow-data-analysis/
+    helm dependency build $directory/forms-flow-web/
+    helm dependency build $directory/forms-flow-documents-api/
 }
 
 displayPrompts() {
-	read -p "Install using forms-flow package registry? (y/n):" is_from_registry
-	read -p "Please enter the domain name (ex: apps.bronze.aot-technologies.com):" domain_name
-	read -p "Please enter the namespace (ex: forms-flow):" namespace
-	read -p "Is this a premium installation? (y/n):" is_premium
-	read -p "Use the latest version release? (y) or stable release (n):" is_latest_release
+    read -p "Install using forms-flow package registry? (y/n):" is_from_registry
+    read -p "Please enter the domain name (ex: apps.bronze.$domain_name):" domain_name
+    read -p "Please enter the namespace (ex: forms-flow):" namespace
+	read -p "Please enter the class name (ex: formsflow):" classname
```

Comment:
What is class name ? If it's ingress classname please make it clear. Also can't we use default if not provided ?

Reply:
yeah, its the ingress class name.. will update that.. if required we can make it default.. the value given in the script is "vault"

Reply:
I think if nothing is provided, it should default to the default ingress class set for the cluster. @amal-k-soman-aot thoughts ?

Reply:
at some point we may not use default class name so that's why we have added the ingress class name. and we have updated it properly while reading it from key board.

Reply:
Ok, please update the question to indicate it's ingress class name. Just asking class name wont be clear for anyone installing this.